app/viewmodels/CashStatementViewModel.scala

```diff
@@ -56,6 +57,22 @@ case class GroupedStatementsByEori(eoriIndex: Int,
 
 object CashStatementViewModel {
 
+  def helpAndSupport(implicit appConfig: FrontendAppConfig, messages: Messages): HtmlFormat.Appendable = {
+
+    val heading = h2Component(
+      id = Some("search-transactions-support-message-heading"),
+      msg = "site.support.heading",
+      classes = "govuk-heading-m govuk-!-padding-top-8")
```

Comment:
I believe we've don't need `govuk-!-padding-top-8` since the layout component already has padding top applied

Reply:
This has been resolved. We need the padding between top text and the "Help and support" text. Adjusted padding to be default spacing across.